app/main/views/users.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from flask import jsonify, request, abort
+from app.main import main
+from app.models import User
+from app import db, encryption
+from sqlalchemy.exc import IntegrityError
+from app.main import helpers
+from app.validation import validate_user_json_or_400, \
+    validate_user_auth_json_or_400
+
+
+@main.route('/users/auth', methods=['POST'])
+def auth_user():
+    json_payload = get_json_from_request('auth_users')
+
+    validate_user_auth_json_or_400(json_payload)
+
+    user = User.query.filter(
+        User.email_address == json_payload['email_address'].lower()).first()
+
+    if user is None:
```

Comment:
I think the general advise is to have the same response for both cases, e.g. "incorrect username and password". This probably doesn't apply here directly, since it's an API and we have a GET user endpoint anyway, but having a single error response would make sure that the frontend apps don't distinguish it either.